libsplinter/src/store/postgres.rs

```diff
@@ -95,7 +95,21 @@ impl StoreFactory for PgStoreFactory {
         Box::new(crate::registry::DieselRegistry::new(self.pool.clone()))
     }
 
-    #[cfg(feature = "authorization")]
+    #[cfg(feature = "role-based-authorization-store-postgres")]
+    fn get_role_based_authorization_store(
+        &self,
+    ) -> Box<dyn crate::rest_api::auth::roles::store::RoleBasedAuthorizationStore> {
+        Box::new(
+            crate::rest_api::auth::roles::store::DieselRoleBasedAuthorizationStore::new(
+                self.pool.clone(),
+            ),
+        )
+    }
+
+    #[cfg(all(
```

Comment:
Suggest removing this unimplemented!() implementation and using generic postgres feature instead of "role-based-authorization-store-postgres".

Reply:
This is part of a pattern we've been using as the postgres stores are not directly tested.  That way, the feature can be stabilized on SQLite before postgres stuff is tested in a running system.  The feature will be removed and replaced when stabilized